lib/resources/example_json/dstu2_examples_medication_administration.rb

```diff
@@ -0,0 +1,378 @@
+module Cerner
+  module Resources
+
+    #These examples are from DEVENG. Update with FHIRPLAY test patients before merging.
+
```

Comment:
Need a multi-ingredient/contained example

Reply:
I've added examples of: Standard administrations, administrations with contained medications, administrations that are not given, and an infusion: https://github.com/cerner/fhir.cerner.com/pull/169/commits/037712040917c1d556c6c8c76363db6eb41412c2#diff-e8b526c8fb4dde5a442627faec99bc0f